### PR TITLE
Creates Ecommerce gem and interface for Ecommerce endpoints

### PR DESCRIPTION
**Pull request by tauil**
Item from [this card](https://trello.com/c/dgjmOPjK/264-16-4-charging-via-ecommerce-3-ecommerce-integration)

### CODE REVIEW

**Comment by rodrigotassinari** [comment]
:+1:

**Comment by tauil** [comment]
This is still in WIP. I'll add some more things here and them you can review.

**Comment by eduardohertz** [comment]
@rtopitt @tauil take a look, guys.

File: ecommerce-client.gemspec [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+require 'ecommerce/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "ecommerce-client"
+  spec.version       = Ecommerce::VERSION
+  spec.authors       = ["Rafael B. Tauil"]
+  spec.email         = ["rafael@tauil.com.br"]
+  spec.summary       = %q{Official Ruby client for the Ecommerce API.}
```
Comment (rodrigotassinari) [inline ecommerce-client.gemspec]: Official Ruby client for **Myfreecomm's Ecommerce API**

File: ecommerce-client.gemspec [hunk 2]
```diff
@@ -0,0 +1,29 @@
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+require 'ecommerce/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "ecommerce-client"
+  spec.version       = Ecommerce::VERSION
+  spec.authors       = ["Rafael B. Tauil"]
+  spec.email         = ["rafael@tauil.com.br"]
```
Comment (rodrigotassinari) [inline ecommerce-client.gemspec]: add eduardo's info as well

**Comment by rodrigotassinari** [comment]
:+1:

**Pull request merged**